portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.time.LocalDateTime;
+
+public class Comment {
+    private String name;
+    private String comment;
+    private LocalDateTime time;
+
+    public Comment(String name, String comment, LocalDateTime time) {
+        this.name = name;
+        this.comment = comment;
+        this.time = time;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
From the server code, you don't use any set methods. So maybe make all variables final and remove all set methods. Making this class immutable.